aiohttp/cookiejar.py

```diff
@@ -471,6 +475,10 @@ def __iter__(self) -> "Iterator[Morsel[str]]":
     def __len__(self) -> int:
         return 0
 
+    @property
+    def quote_cookie(self) -> bool:
+        return True
```

Comment:
I'm not sure if `quote_cookie` in dummy cookie jar makes sense, for it does not handle any cookies. I let it return True simply because it returns `SimpleCookie` in `filter_cookies`.